pristine-zen-garden.html

```diff
@@ -0,0 +1,283 @@
+    <link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Dave Shea">
+    <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+    <meta name="robots" content="all">
+
+</head>
+
+<body id="css-zen-garden">
+    <div class="page-wrapper">
+        <section class="intro" id="zen-intro">
+            <header role="banner">
+                <h1>CSS Zen Garden</h1>
+                <h2>
+                    The Beauty of
+                    <abbr title="Cascading Style Sheets">CSS</abbr> Design
+                </h2>
+            </header>
+            <div class="summary" id="zen-summary" role="article">
+                <p>
+                    A demonstration of what can be accomplished through
+                    <abbr title="Cascading Style Sheets">CSS</abbr>-based design. Select any style sheet from the list
+                    to load it into this page.
+                </p>
+                <p>
+                    Download the example
+                    <a href="/examples/index" title="This page's source HTML code, not to be modified.">html file</a> and
+                    <a href="/examples/style.css" title="This page's sample CSS, the file you may modify.">css file</a>
+                </p>
+            </div>
+            <div class="preamble" id="zen-preamble" role="article">
+                <h3>The Road to Enlightenment</h3>
+                <p>
+                    Littering a dark and dreary road lay the past relics of browser-specific tags, incompatible
+                    <abbr title="Document Object Model">DOM</abbr>s, broken
+                    <abbr title="Cascading Style Sheets">CSS</abbr> support, and abandoned browsers.
+                </p>
+                <p>
+                    We must clear the mind of the past. Web enlightenment has been achieved thanks to
+                    the tireless efforts of folk like the
+                    <abbr title="World Wide Web Consortium">W3C</abbr>,
+                    <abbr title="Web Standards Project">WaSP</abbr>, and the major browser creators.
+                </p>
+                <p>
+                    The CSS Zen Garden invites you to relax and meditate on the important
+                    lessons of the masters. Begin to see with clarity. Learn to use
+                    the time-honored techniques in new and invigorating fashion.
+                    Become one with the web.
+                </p>
+            </div>
+        </section>
+
+        <div class="main supporting" id="zen-supporting" role="main">
+            <div class="explanation" id="zen-explanation" role="article">
+                <h3>So What is This About?</h3>
+                <p>There is a continuing need to show the power of
```

Comment:
<p on the wrong line

Reply:
I don't see you comment anything, but I know what you meant. Thanks!